templates/music/album_detail.html

```diff
@@ -15,16 +15,27 @@
         <h4>Release date: {{ album.release_date }}</h4>
     {% endif %}
     {% if album.albumbio %}
-        <p>{{ album.albumbio.bio|markdown }}</p>
+        <h4>About</h4>
+        <div class="break-small">
+            <hr>
+        </div>
+        <div class="album-bio">
+            <ol class="album-bio">
```

Comment:
It does not make sense to wrap this section in an `<ol>`. The bio markdown can generate `<ol>`s.